packages/core/src/api.ts

```diff
@@ -99,45 +97,6 @@ export class API {
     };
   }
 
-  /** Returns the url to the report dialog endpoint. */
-  public getReportDialogEndpoint(
-    dialogOptions: {
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      [key: string]: any;
-      user?: { name?: string; email?: string };
-    } = {},
-  ): string {
-    const dsn = this.getDsn();
-    const endpoint = `${this.getBaseApiEndpoint()}embed/error-page/`;
-
-    const encodedOptions = [];
-    encodedOptions.push(`dsn=${dsn.toString()}`);
-    for (const key in dialogOptions) {
-      if (key === 'dsn') {
-        continue;
-      }
-
-      if (key === 'user') {
-        if (!dialogOptions.user) {
-          continue;
-        }
-        if (dialogOptions.user.name) {
-          encodedOptions.push(`name=${encodeURIComponent(dialogOptions.user.name)}`);
-        }
-        if (dialogOptions.user.email) {
-          encodedOptions.push(`email=${encodeURIComponent(dialogOptions.user.email)}`);
-        }
-      } else {
-        encodedOptions.push(`${encodeURIComponent(key)}=${encodeURIComponent(dialogOptions[key] as string)}`);
-      }
-    }
-    if (encodedOptions.length) {
-      return `${endpoint}?${encodedOptions.join('&')}`;
-    }
-
-    return endpoint;
```

Comment:
I removed this logic because `encodedOptions.push(`dsn=${dsn.toString()}`);` meant that the `return endpoint;` would never be hit